### ISSUE TITLE
无法找到错误格式

### ISSUE THREAD

**Issue by Nitmi** [Jinjiang/zhlint#168]
安装之后直接使用，没有对配置文件做任何更改。

```bash
❯ cat foo.md
# test

中文English

D:\
❯ zhlint .\foo.md
Config file "D:\.zhlintrc" does not exist. Will proceed as default.
Global ignored cases file "D:\.zhlintignore" does not exist. Will proceed as none.
Global ignored cases file "D:\.zhlintcaseignore" does not exist. Will proceed as none.
No error found.

D:\
❯ zhlint .\foo.md --fix
Config file "D:\.zhlintrc" does not exist. Will proceed as default.
Global ignored cases file "D:\.zhlintignore" does not exist. Will proceed as none.
Global ignored cases file "D:\.zhlintcaseignore" does not exist. Will proceed as none.
No error found.
```

难道没有默认规则？
